src/components/headerNav.js

```diff
@@ -0,0 +1,102 @@
+
+import React,{ Component } from 'react'
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+import { push } from 'react-router-redux'
+import { Route } from 'react-router-dom'
+import AppBar from 'material-ui/AppBar';
+import IconButton from 'material-ui/IconButton';
+import IconMenu from 'material-ui/IconMenu';
+import MenuItem from 'material-ui/MenuItem';
+import FlatButton from 'material-ui/FlatButton';
+
+import Avatar from 'material-ui/Avatar';
+import RaisedButton from 'material-ui/RaisedButton';
+import '../css/index.css'
```

Comment:
why?

Reply:
Do you have knowledge of material-ui?

Reply:
index.css is supposed to propagate, you already imported it in App.js, and it is the same index.css file. Was not referring to the material components...